wordpress-mobile/WordPress-iOS
# Update terms in code to use more inclusive language

Fixes #15522

- Replaced `whitelist` references with `allowlist`
- Replaced `blacklist` references with `blocklist`
- Created a new xcdatamodel to rename some of `BlogSettings` entity properties that included those references
- Renamed `jetpack_whitelisted_ips_viewed` event to `jetpack_allowlisted_ips_viewed`
- Renamed `jetpack_whitelisted_ips_changed` event to `jetpack_allowlisted_ips_changed `

To test:

- Check Core Data migration: Install the app on top of an old version and check if it's launching without errors
- Update `Blocklist` and `Automatically Approve` under `Settings` -> `Discussions` and check if they are working correctly
- From a Jetpack connected site update `Allowlisted IP addresses` under `Jetpack Settings` and check if it's working correctly

PR submission checklist:
